draft-bormann-gendispatch-with-expert-review.md

```diff
@@ -90,13 +96,20 @@ requirements) that would have been pointed out by the expert.
 
 [^a3-] {{Sections 2 and 3 of RFC7120@BCP100}} [^a4-]
 
+[^a5-]
 
 # Augmented Registration Policies {#augment}
 
 For each of the well-known policies defined in {{Sections 4.7, 4.8,
 4.9, and 4.10 of RFC8126@BCP26}}, this document adds a parallel _augmented
 policy_ that also specifies involving a Designated Expert.
 
+As long as his has not happened, authors of specifications that want
```

Comment:
OLD:
As long as his has not happened, authors ...

NEW:
For the period of time while {{RFC8126@BCP26}} has not been updated to include the augmented registration policies, authors ...